packages/js/README.md

```diff
@@ -238,6 +238,39 @@ use `client.getIsRegistered` method
 client.getIsRegistered().then(isRegistered => {...})
 ```
 
+### Anonymous Login for AI assistant
+
+The TelnyxRTC client can be used with an AI assistant by using the `anonymous_login` method. This allows you to authenticate without a SIP connection, making it easier to integrate with AI assistants.
+
+```js
+const client = new TelnyxRTC({
+  anonymous_login: {
+    target_id: 'YOUR_TARGET_ID',
+    target_type: 'ai_assistant',
```

Comment:
could you update the class docs to include possible options for this? I understood this is just for type "ai_assistant" now but it'd be good to know more details about this field and target_id as the docs I shared above only include a short headline of each